types/validator_set.go

```diff
@@ -19,7 +19,10 @@ import (
 // (Proof of 1 is tricky, left to the reader).
 // It could be higher, but this is sufficiently large for our purposes,
 // and leaves room for defensive purposes.
-const MaxTotalVotingPower = int64(math.MaxInt64) / 8
+const (
+	MaxTotalVotingPower = int64(math.MaxInt64) / 8
+	K                   = 2
```

Comment:
What is K?

Reply:
it's 2 :)...changed to a nice long name and added comment explaining.